nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/DepthwiseConv2DBp.java

```diff
@@ -0,0 +1,241 @@
+        }
+
+        try {
+            val t = config.getValue(property);
+            return t;
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Map<String, Object> propertiesForFunction() {
+        if (config == null && !iArguments.isEmpty()) {
+            config = Conv2DConfig.builder()
+                    .kH(iArguments.get(0))
+                    .kW(iArguments.get(1))
+                    .sH(iArguments.get(2))
+                    .sW(iArguments.get(3))
+                    .pH(iArguments.get(4))
+                    .pW(iArguments.get(5))
+                    .dH(iArguments.get(6))
+                    .dW(iArguments.get(7))
+                    .isSameMode(iArguments.get(8) == 1)
+                    .dataFormat(iArguments.get(9) == 1 ? Conv2DConfig.NHWC : Conv2DConfig.NCHW)
+                    .build();
+        }
+        return config.toProperties();
+    }
+
+
+    @Override
+    public boolean isConfigProperties() {
+        return true;
+    }
+
+    @Override
+    public String configFieldName() {
+        return "config";
+    }
+
+
+    @Override
+    public Map<String, Map<String, AttributeAdapter>> attributeAdaptersForFunction() {
+        Map<String, Map<String, AttributeAdapter>> ret = new HashMap<>();
+        Map<String, AttributeAdapter> tfMappings = new LinkedHashMap<>();
+        val fields = DifferentialFunctionClassHolder.getInstance().getFieldsForFunction(this);
+
+
+        //TF uses [kH, kW, inC, outC] always for weights
+        tfMappings.put("kH", new NDArrayShapeAdapter(0));
+        tfMappings.put("kW", new NDArrayShapeAdapter(1));
+        tfMappings.put("sH", new ConditionalFieldValueIntIndexArrayAdapter("NCHW", 2, 1, fields.get("dataFormat")));
+        tfMappings.put("sW", new ConditionalFieldValueIntIndexArrayAdapter("NCHW", 3, 2, fields.get("dataFormat")));
+        tfMappings.put("dH", new ConditionalFieldValueIntIndexArrayAdapter("NCHW", 2, 1, fields.get("dataFormat")));
+        tfMappings.put("dW", new ConditionalFieldValueIntIndexArrayAdapter("NCHW", 3, 2, fields.get("dataFormat")));
+        tfMappings.put("isSameMode", new StringEqualsAdapter("SAME"));
+
+
+        Map<String, AttributeAdapter> onnxMappings = new HashMap<>();
+        onnxMappings.put("kH", new SizeThresholdIntArrayIntIndexAdpater(0, 2, 0));
+        onnxMappings.put("kW", new SizeThresholdIntArrayIntIndexAdpater(1, 2, 0));
+        onnxMappings.put("dH", new SizeThresholdIntArrayIntIndexAdpater(0, 2, 0));
+        onnxMappings.put("dW", new SizeThresholdIntArrayIntIndexAdpater(1, 2, 0));
+        onnxMappings.put("sH", new SizeThresholdIntArrayIntIndexAdpater(0, 2, 0));
+        onnxMappings.put("sW", new SizeThresholdIntArrayIntIndexAdpater(1, 2, 0));
+        onnxMappings.put("isSameMode", new StringEqualsAdapter("SAME"));
+
+        return ret;
+    }
+
+    @Override
+    public Map<String, Map<String, PropertyMapping>> mappingsForFunction() {
```

Comment:
Cut both the attributeAdaptersForFunction and mappingsForFunction.
These are for tensorflow/onnx import only.
None of the backprop operations should have import support (with like 2 exceptions in the whole codebase).